### PR TITLE
Fix test_sip_approx test for gwcs>=0.18.2

### PR DESCRIPTION
**Pull request by mcara**
Updates/Fixes test_sip_approx test for gwcs>=0.18.2

**Checklist for maintainers**
- [ ] added entry in `CHANGES.rst` within the relevant release section
- [x] updated or added relevant tests
- [ ] updated relevant documentation
- [x] added relevant milestone
- [x] added relevant label(s)
- [ ] ran regression tests, post a link to the Jenkins job below.
      [How to run regression tests on a PR](https://github.com/spacetelescope/jwst/wiki/Running-Regression-Tests-Against-PR-Branches)
- [ ] Make sure the JIRA ticket is [resolved properly](https://github.com/spacetelescope/jwst/wiki/How-to-resolve-JIRA-issues)

### CODE REVIEW

**Comment by mcara** [comment]
To be merged after PyPi release 0.18.2cof `gwcs` is published

File: jwst/assign_wcs/tests/test_wcs.py [hunk 1]
```diff
@@ -267,6 +261,7 @@ def test_sip_approx(tmpdir):
         if 'ctype' in key:
             assert true_wcs[key] == wcs_info[key]
         else:
+            print(f"{key}:  {wcs_info[key]}")
```
Comment (nden) [inline jwst/assign_wcs/tests/test_wcs.py:264]: Is this `print` intentional?

**Review by nden — APPROVED** [review]
LGTM.
I left one minor comment. Needs a rebase

File: jwst/assign_wcs/tests/test_wcs.py (hunk 1 above)
Comment (mcara) [reply jwst/assign_wcs/tests/test_wcs.py:264]: No.

**Review by hbushouse — APPROVED** [review]
Looks OK to me too, modulo the question about the print statement.

**Comment by mcara** [comment]
removed the `print` statement and rebased.

**Comment by hbushouse** [comment]
It looks like gwcs 0.18.2 has been release on pypi, so I assume we can merge this when the CI tests have finished?

**Comment by mcara** [comment]
Yes

**Pull request merged**